firmware/powerboard2/code/app/RTOS/RTOS_componentSpecific.c

```diff
@@ -49,5 +51,9 @@ void RTOS_run100ms(void)
 
 void RTOS_run1000ms(void)
 {
-
+	uint8_t data[] = {0x1, 0b0100000, 0b0};
+	uint8_t* a = &data[0];
+	I2C_send(I2C_CHANNEL_VBATT, a, 2);
+	while (!is_idle()){}
```

Comment:
How long does this loop last for?

Reply:
Not very long. According to what I saw on logic analyzer, right after the send process stop the read process started